doc/api.md

```diff
@@ -153,10 +180,10 @@ Returns:
 -   400 MISSING_HEADERS if any of the mandatory headers is not present.
 -   500 SERVER ERROR if there was any error not contemplated above.
 
-##### PUT /iot/services
+##### PUT /iot/services or PUT /iot/configGroups
```

Comment:
```suggestion
##### PUT /iot/configGroups or PUT /iot/services
```

(and similar change in other heading in this document)

Reply:
Fixed in 424d59d